tests/integration/helpers/app-version-test.js

```diff
@@ -11,15 +10,13 @@ test('it displays only app version', function(assert) {
 
   this.render(hbs`{{app-version hideSha=true}}`);
 
-  assert.ok(this.$().text().match(versionRegExp), 'Displays version.');
-  assert.ok(!this.$().text().match(shaRegExp), 'Does not display git sha.');
+  assert.equal(this.$().text(), '1.2.3', 'Displays version.');
 });
 
 test('it displays only git sha', function(assert) {
   assert.expect(2);
 
   this.render(hbs`{{app-version hideVersion=true}}`);
 
-  assert.ok(this.$().text().match(shaRegExp), 'Displays git sha.');
-  assert.ok(!this.$().text().match(versionRegExp), 'Does not display version.');
+  assert.equal(this.$().text(), 'deadbeef', 'Displays git sha.');
```

Comment:
To make obvious what is missing. Just like in line 13 where I check for version 1.2.3
The values returned by the app currently are those of ember-cli-app-version which of course mean it is no possible to test for exact values.
But I'll have a look at your suggestions in the pull request on how to stub these values in the config and then test here for the stubbed value.